src/pages/GE_TokenizeCard.page

```diff
@@ -10,6 +10,12 @@
             <div id="payments"></div>
         </form>
     </article>
+    
+    <!--TODO - remove once elevate can process the gateway id-->
+    <!-- for QE: to display visually the gateway id when it's passed to tokenize method -->
+    <article class='slds-scope'>
+        <div><small><span id="gatewayInfo"></span></small></div>
```

Comment:
@vmartinov-sforg Was there supposed to be content within this tag? I couldn't find it initially and I realized it's because there's nothing actually rendered within the span or div tag!

Reply:
The text of the span is set at run-time to display the gateway is when the button is clicked. @ptewson-sfdo